changelog/@unreleased/pr-4353.v2.yml

```diff
@@ -0,0 +1,20 @@
+type: break
+break:
+  description: |-
+    Bump Postgres min version to 9.6
+
+    **Goals (and why)**:
+    Stop supporting old versions. Enable using new Postgres features.
+
+    **Implementation Description (bullets)**:
+
+    **Testing (What was existing testing like?  What have you done to improve it?)**:
+
+    **Concerns (what feedback would you like?)**:
+
+    **Where should we start reviewing?**:
+
+    **Priority (whenever / two weeks / yesterday)**:
```

Comment:
ah, not sure we want the whole changelog template in release notes. I'll fix this